spreg/twosls_regimes.py

```diff
@@ -497,6 +501,7 @@ def _test():
     _test()
     import numpy as np
     import libpysal
+    from libpysal.examples import load_example
```

Comment:
is `load_example` being used here?

Reply:
it is.  the NAT example data set

Reply:
I just added the missing line! Thank you for spotting this!